src/core/common/tlvs.hpp

```diff
@@ -585,7 +585,7 @@ class Tlv
      * @retval kErrorNoBufs   Insufficient available buffers to grow the message.
      *
      */
-    static Error AppendTlv(Message &aMessage, uint8_t aType, const void *aValue, uint8_t aLength);
+    static Error AppendTlv(Message &aMessage, uint8_t aType, const void *aValue, uint16_t aLength);
```

Comment:
Thanks for changing this. 👍 

Suggest updating documentation  to mention the change in this method? (that it will append the TLV as an extended TLV if it does not fit in regular TLV).

Reply:
I have updated the doc.